src/elements/icon-tile/src/index.tsx

```diff
@@ -3,20 +3,38 @@
 import * as React from 'react'
 import { jsx, Styled } from 'theme-ui'
 
+enum DisplayVariant {
```

Comment:
export the enum so it can be used elsewhere

Reply:
updated